external/CMakeLists.txt

```diff
@@ -27,26 +27,26 @@ function(pop_variable var)
 endfunction()
 
 if (DEFINED SPIRV-Headers_SOURCE_DIR)
-  # This allows flexible position of the SPIRV-Headers repo.
-  set(SPIRV_HEADER_DIR ${SPIRV-Headers_SOURCE_DIR})
+  add_subdirectory(${SPIRV-Headers_SOURCE_DIR})
```

Comment:
Line 43 in the old implementation stops `add_subdirectory` from being called when `SPIRV-Headers_SOURCE_DIR` is set. This was used to identify if the subdirectory was already added in the parent project.